tests/AppBundle/Controller/MembershipControllerTest.php

```diff
@@ -283,12 +279,86 @@ public function testPinInterestsPersistsInterestsForNonActivatedAdherent()
 
         $this->assertClientIsRedirectedTo('/inscription/choisir-des-comites', $this->client);
 
-        /* @var Adherent $adherent */
+        /** @var Adherent $adherent */
         $adherent = $this->getAdherentRepository()->findByEmail('michelle.dufour@example.ch');
 
         $this->assertSame(array_values($chosenInterests), $adherent->getInterests());
     }
 
+    public function testChooseNearbyCommitteeWithoutNewAdherentId()
+    {
+        $this->client->request(Request::METHOD_GET, '/inscription/choisir-des-comites');
+
+        $this->assertResponseStatusCode(Response::HTTP_NOT_FOUND, $this->client->getResponse());
+    }
+
+    public function testChooseNearbyCommitteeWithWrongNewAdherentId()
+    {
+        $this->client->getContainer()->get('session')->set(MembershipUtils::NEW_ADHERENT_ID, 'wrong id');
+        $this->client->request(Request::METHOD_GET, '/inscription/choisir-des-comites');
+
+        $this->assertResponseStatusCode(Response::HTTP_NOT_FOUND, $this->client->getResponse());
+    }
+
+    public function testChooseNearbyCommittee()
+    {
+        $this->client->getContainer()->get('session')->set(MembershipUtils::NEW_ADHERENT_ID, 1);
+
+        $crawler = $this->client->request(Request::METHOD_GET, '/inscription/choisir-des-comites');
+
+        $boxPattern = '#app_membership_choose_nearby_committee_committees > div';
+
+        $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
+        $this->assertCount(3, $boxes = $crawler->filter($boxPattern));
+
+        /** @var Committee[] $committees */
+        $committees = $this->getCommitteeRepository()->findBy([], ['id' => 'desc'], 3);
```

Comment:
For this one, this will be the case in the next issue. Just for test purpose here.